### PR TITLE
Add YouTube Video Embeds to Docs Site

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds Youtube Video Embeds to relevant spots on the Tanssi Docs site. 

### Checklist

- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/interact/ethereum-api/libraries/ethersjs.md [hunk 1]
```diff
@@ -74,6 +74,9 @@ Save this code snippet as you'll need it for the scripts that are used in the fo
 
 During this section, you'll be creating a couple of scripts. The first one will be to check the balances of your accounts before trying to send a transaction. The second script will actually send the transaction.
 
+<style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/bXtknNifO78?si=T6VtPQyS6q01RC3Q' frameborder='0' allowfullscreen></iframe></div>
```
Comment (eshaben) [inline builders/interact/ethereum-api/libraries/ethersjs.md:77]: The placement of this one should maybe be after the following sentence

File: builders/interact/ethereum-api/libraries/ethersjs.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ description: Learn how to use the Ethereum EtherJS Library to send transactions
...
 
-In this guide, you'll learn how to use setup the Ethers.js library for your Tanssi EVM ContainerChain. Next, to showcase the library in action, you'll use Ethers.js to send a transaction and deploy a contract on a Tanssi EVM ContainerChain running in Tanssi's [Dancebox](/builders/tanssi-network/networks/dancebox){target=_blank} TestNet. This guide can be adapted for your own Tanssi EVM ContainerChain by simply changing the endpoint.
+In this guide, you'll learn how to use setup the Ethers.js library for your Tanssi EVM ContainerChain. Next, to showcase the library in action, you'll use Ethers.js to send a transaction and deploy a contract on a Tanssi EVM ContainerChain running in Tanssi's [Dancebox](/builders/tanssi-network/networks/dancebox){target=_blank} TestNet. This guide can be adapted for your own Tanssi EVM ContainerChain by simply changing the endpoint. If you prefer following video tutorials, you'll find a corresponding video tutorial for [Sending Transactions with Ethers.js](#send-a-transaction) and [Deploying Contracts with Ethers.js](#deploy-a-contract)
```
Comment (eshaben) [inline builders/interact/ethereum-api/libraries/ethersjs.md:12]:
```suggestion
In this guide, you'll learn how to use the Ethers.js library for your Tanssi EVM ContainerChain. Next, to showcase the library in action, you'll use Ethers.js to send a transaction and deploy a contract on a Tanssi EVM ContainerChain running in Tanssi's [Dancebox](/builders/tanssi-network/networks/dancebox){target=_blank} TestNet. This guide can be adapted for your own Tanssi EVM ContainerChain by simply changing the endpoint. 

If you prefer to follow video tutorials, you'll find a corresponding video tutorial for [Sending Transactions with Ethers.js](#send-a-transaction) and [Deploying Contracts with Ethers.js](#deploy-a-contract).
```

**Comment by albertov19** [comment]
@eshaben @themacexpert this looks good to me, another option would be to have it at the top just below the `#` title so that it is shown as soon as the page is opened?

**Comment by themacexpert** [comment]
> @eshaben @themacexpert this looks good to me, another option would be to have it at the top just below the `#` title so that it is shown as soon as the page is opened?

I agree that is the best place. In the case of the Ethers.js tutorial there are two videos, 1 for transactions and 1 for contracts. Currently they are beneath the subheadings for each, but we could place one of those at the top for variety maybe or just to draw people in?

**Comment by albertov19** [comment]
> > @eshaben @themacexpert this looks good to me, another option would be to have it at the top just below the `#` title so that it is shown as soon as the page is opened?
> 
> I agree that is the best place. In the case of the Ethers.js tutorial there are two videos, 1 for transactions and 1 for contracts. Currently they are beneath the subheadings for each, but we could place one of those at the top for variety maybe or just to draw people in? @albertov19

Umm no personal preference. Is there a way to embed them side to side depending on the screen width? If not, I'll let @eshaben decide how she think it looks better 😄

**Comment by eshaben** [comment]
> Is there a way to embed them side to side depending on the screen width? 

Ah yeah, that is a good idea

**Comment by themacexpert** [comment]
Made the changes as discussed!

**Comment by albertov19** [comment]
@themacexpert I thought we discussed putting it below the `#` title above the introduction? Also only the Ethers one has captions I would include captions in all or none 😄

**Comment by themacexpert** [comment]
@albertov19 All set now! I also updated the other videos so we put them all right below the # consistently. Regarding the captions I believe that isn't video captions that's just the subtext below the video to show one is the Transaction one and which one is the Deploy Contracts one.

File: builders/interact/ethereum-api/libraries/ethersjs.md [hunk 3]
```diff
@@ -5,11 +5,16 @@ description: Learn how to use the Ethereum EtherJS Library to send transactions
...
 
-In this guide, you'll learn how to use setup the Ethers.js library for your Tanssi EVM ContainerChain. Next, to showcase the library in action, you'll use Ethers.js to send a transaction and deploy a contract on a Tanssi EVM ContainerChain running in Tanssi's [Dancebox](/builders/tanssi-network/networks/dancebox){target=_blank} TestNet. This guide can be adapted for your own Tanssi EVM ContainerChain by simply changing the endpoint.
+In this guide, you'll learn how to use the Ethers.js library for your Tanssi EVM ContainerChain. Next, to showcase the library in action, you'll use Ethers.js to send a transaction and deploy a contract on a Tanssi EVM ContainerChain running in Tanssi's [Dancebox](/builders/tanssi-network/networks/dancebox){target=_blank} TestNet. This guide can be adapted for your own Tanssi EVM ContainerChain by simply changing the endpoint. 
+
+If you prefer to follow video tutorials, you'll find corresponding video tutorials below for [Sending Transactions with Ethers.js](#send-a-transaction) and [Deploying Contracts with Ethers.js](#deploy-a-contract).
```
Comment (eshaben) [inline builders/interact/ethereum-api/libraries/ethersjs.md:16]: This just needs to be updated now 🙂

File: builders/interact/ethereum-api/libraries/ethersjs.md [hunk 4]
```diff
@@ -5,11 +5,16 @@ description: Learn how to use the Ethereum EtherJS Library to send transactions
 
 # Ethers.js JavaScript Library
 
+<style>.video-container{display:flex;justify-content:space-between;max-width:1000px;margin:0 auto;}.video-column{width:49%;}.embed-container{position:relative;padding-bottom:56.25%;height:0;overflow:hidden;max-width:100%;margin-bottom:10px;}.embed-container iframe,.embed-container object,.embed-container embed{position:absolute;top:0;left:0;width:100%;height:100%;}.caption{font-family:Open Sans,sans-serif;font-size:0.9em;color:rgba(170,170,170,1);font-style:italic;letter-spacing:0px;position:relative;}</style><div class="video-container"><div class="video-column"><div class='embed-container'><iframe src='https://www.youtube.com/embed/bXtknNifO78?si=Hjiah5yhKvBcsE7f' frameborder='0' allowfullscreen></iframe></div><div class="caption">How to Send a Simple Transaction</div></div><div class="video-column"><div class='embed-container'><iframe src='https://www.youtube.com/embed/m9iVeCP7owI?si=krm6z9AsFCDrjlJh' frameborder='0' allowfullscreen></iframe></div><div class="caption">How to Deploy a Contract</div></div></div>
```
Comment (eshaben) [inline builders/interact/ethereum-api/libraries/ethersjs.md:8]: I'm sorry for the back and forth! I just re-read Alberto's last comment about the captions, and I agree. I don't think we need the caption or text underneath the videos, but I'm also not opposed to them. So, I would agree with Alberto and say either add the text beneath all of the videos or remove them here

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**